dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPI.java

```diff
@@ -1115,6 +1115,13 @@ SystemActionWorkflowActionMapping mapSystemActionToWorkflowActionForWorkflowSche
 	 */
 	long countAllSchemasSubActions(User user) throws DotDataException, DotSecurityException;
 
+	/**
+	 * Return the count of Unique SubAction in all Action
+	 *
+	 * @return
+	 */
+	long countAllSchemasUniqueSubActions(User user) throws DotDataException, DotSecurityException;
```

Comment:
Change to: Returns the count of unique subactions in all Workflow actions

Reply:
done https://github.com/dotCMS/core/pull/26869/commits/e997967ea222c2992306f6c3c25dc24d2c006b53#diff-d7c7d1cb2ec9c3b8af52d37d58c0b0ac6705f249676a0f454b554c2690311864R1119